docs/stdlib.md

```diff
@@ -939,14 +939,12 @@ bpftrace doesn’t read past the length of the shortest string.
 
 
 ### strerror
-- `strerror_t strerror(int error)`
 
 Convert errno code to string.
```

Comment:
let's keep the variant

Reply:
we already have a bunch of "made-up" types in our docs `kstack_t`, `cgroup_path_t `, etc.

Reply:
I mean that there is no much such type as `strerror_t`, this will return a `string[?]` and I'm not sure of the right way to express this. I don't think anyone cared about the type before (you couldn't do anything with it), and I don't think they care now, so I'm just trying to understand `variant` for this kind of case.